daemons/gptp/gptp_cfg.ini

```diff
@@ -31,19 +31,11 @@ syncReceiptThresh = 8
 
 [eth]
 
-# PHY delay GB TX in nanoseconds
+# PHY delay GB TX/RX in nanoseconds
```

Comment:
I've push changes that keep the old format and add a new format:
     phy_delay = LINKSPEED_1G 184 382

The there are some predefined speeds that can be used like above or the speed can be entered numerically.